components/cardOurPartners/ourPartnersCard.js

```diff
@@ -0,0 +1,10 @@
+import styles from "./cardOurPartners.module.css"
+
+export default function OurPartnersCard({ logo }) {
+   return <>
+      <div>
+         <img className={styles.ourPartnersItemImage} src={logo} alt='our partner' />
```

Comment:
This `alt="..."` text is not providing any extra info for the sighted users, so you can either change it or remove it.
- More info on alt tag: https://www.w3.org/WAI/tutorials/images/
- Videos about Accessibility testing, and how people use assistive technologies:
  - Short intro video - https://www.youtube.com/watch?v=HE2R86EZPMA
  - Longer and detailed video from a non-sighted user - https://www.youtube.com/watch?v=OUDV1gqs9GA